aces/Loris
# [Dashboard] Fix discrepancies for "Violated Scans" in Dashboard

## Brief summary of changes
-Fixes #5289 (MRI Violations Discrepancies)


#### Testing instructions (if applicable)

1. Before checking out this PR, make a note on the count on mri violated count on dashboard and as well as in mri violations module. (There will be mismatch in the count)
2. Checkout to this branch and see if the MRI violation count in Dashboard matches with Unresolved violations in MRI violations module,


NOTE: The database query used to fetch mri violations log in mri violations module is  moved to a class constant and referred to in  `getWidgets()` function to fix the discrepancies.
I suggest this to be tested in project having an actual imaging dataset.
